scraper/scraper_impl/capes_scraper.py

```diff
@@ -88,7 +89,7 @@ def iter_departments_by_thread(self, thread_id, num_threads):
 
         # Iterate through each department that the thread is assigned to
         for counter in range(thread_id, len(self.departments), num_threads):
-            # Exit if any part of the scraper has crashed 
+            # Exit if any part of the scraper_impl has crashed
```

Comment:
will fix this!

Reply:
how should we do that? catch exception?

Reply:
oh no I meant the fix the comment, sorry!